src/fileviewer.cpp

```diff
@@ -392,14 +394,41 @@ void FileViewer::ShowError(const char *icon, const wxString& msg, const wxString
 
 void FileViewer::OnChoice(wxCommandEvent &event)
 {
-    SelectReference(m_references[event.GetSelection()]);
+    m_chosenRefIndex = event.GetSelection();
+    SelectReference(m_references[m_chosenRefIndex]);
 }
 
-void FileViewer::OnEditFile(wxCommandEvent&)
+void FileViewer::OnEditFile(wxCommandEvent &)
```

Comment:
Superfluous unrelated change

Reply:
Yes, It is not necessary and removed. It was a left over my try-and-error.